web/modules/custom/mymodule/src/Form/SimpleForm.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Drupal\mymodule\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Locale\CountryManager as CountryManager;
+use Drupal\Core\Messenger\MessengerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Simpleform class using simple form api.
+ */
+class SimpleForm extends FormBase {
+
+  /**
+   * Messenger variable.
+   *
+   * @var object
+   */
+  protected $messenger;
+
+  /**
+   * Call the messenger using container interface for DI.
+   *
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *   Call the container Interface.
+   *
+   * @return object
+   *   It return the messenger functions.
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('messenger')
+    );
+
+  }
+
+  /**
+   * Construct function to call messenger.
+   *
+   * @param \Drupal\Core\Messenger\MessengerInterface $messenger
+   *   Call the messenger interface.
+   */
+  public function __construct(MessengerInterface $messenger) {
+    $this->messenger = $messenger;
+  }
+
+  /**
+   * Get the formid function.
+   *
+   * @return string
+   *   It return the formid.
+   */
+  public function getFormId() {
+    return 'Simple_Form';
+  }
+
+  /**
+   * Buildform function to create a form fields.
+   *
+   * @param array $form
+   *   It return the form fields as array.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   It return the stored values of form.
+   *
+   * @return array
+   *   Form is returned as array.
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+
+    $countries = CountryManager::getStandardList();
+    $country_array[''] = 'select';
+    foreach ($countries as $key => $value) {
+      $country_name = (string) $value;
+      $country_array[$country_name] = $country_name;
+    }
+    $formValues = $form_state->get('storelocations');
```

Comment:
The correct name of the variable would be - `$num_locations`. Also, we should only follow one pattern for naming variables - it could either be camelcase or all lowercase. In PHP preferred is `lowercase`

Reply:
Ok. changed the variables as you have suggested